templates/index.html

```diff
@@ -29,44 +22,102 @@ <h1 class="text-5xl md:text-6xl xl:text-7xl font-bold tracking-tight mb-12">
+				</svg>
 			</div>
+<<<<<<< HEAD
+			<!--community/{{post.community_name}}/{{post.post_id}}-->
+			<a href="#">
+				<!-- POST CONTENT (PREVIEW)-->
+				<div class="ml-4">
+					<span class="inline-block align-text-top text-base text-gray-900 dark:text-gray-800">g/
+						| Posted by u/{{post.author}} {{post.date_posted}}
+					</span>
+					<h2 class="flex text-2xl align-top font-bold mb-2 text-gray-800">{{post.title}}</h2>
+					<p class="text-gray-700">{{post.content}}</p>
+				</div>
+			</a>
+=======
 			<a href="/community/{{post.community_name}}/{{post.post_id}}">
 			<!-- POST CONTENT (PREVIEW)-->
 			<div class="ml-4">
-				<span class="inline-block align-text-top text-base text-gray-900 dark:text-gray-800"
-					>g/{{post.community_name}} | Posted by u/{{post.author}} {{post.date_posted}}
+				<span class="inline-block align-text-top text-base text-gray-900 dark:text-gray-800">g/Overwatch |
+					Posted by xXFuckHealingxX 1 hour ago
+				</span>
+				<h2 class="flex text-2xl align-top font-bold mb-2 text-gray-800">I really hate the new OW2 Update</h2>
+				<p class="text-gray-700">Cannot believe I have to actually heal as moira...</p>
+			</div>
+		</div>
+	</div>
+
+	<div class="slide-top p-10">
+		<div class="flex flex-row bg-white p-6 rounded-lg shadow-lg">
+			<!-- UPVOTE -->
+			<div>
+				<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5"
+					stroke="currentColor" class="w-6 h-6">
+					<path stroke-linecap="round" stroke-linejoin="round"
+						d="M15 11.25l-3-3m0 0l-3 3m3-3v7.5M21 12a9 9 0 11-18 0 9 9 0 0118 0z" />
+				</svg>
+
+				<p class="text-center">0</p>
+			</div>
+			<!-- POST CONTENT (PREVIEW)-->
+			<div class="ml-4">
+				<span class="inline-block align-text-top text-base text-gray-900 dark:text-gray-800">g/Overwatch |
+					Posted by xXFuckHealingxX 1 hour ago
+				</span>
+				<h2 class="flex text-2xl align-top font-bold mb-2 text-gray-800">I really hate the new OW2 Update</h2>
+				<p class="text-gray-700">Cannot believe I have to actually heal as moira...</p>
+			</div>
+		</div>
+	</div>
+
+	<div class="slide-top p-10">
+		<div class="flex flex-row bg-white p-6 rounded-lg shadow-lg">
+			<!-- UPVOTE -->
+			<div>
+				<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5"
+					stroke="currentColor" class="w-6 h-6">
+					<path stroke-linecap="round" stroke-linejoin="round"
+						d="M15 11.25l-3-3m0 0l-3 3m3-3v7.5M21 12a9 9 0 11-18 0 9 9 0 0118 0z" />
+				</svg>
+
+				<p class="text-center">0</p>
+			</div>
+			<!-- POST CONTENT (PREVIEW)-->
+			<div class="ml-4">
+				<span class="inline-block align-text-top text-base text-gray-900 dark:text-gray-800">g/Overwatch |
+					Posted by xXFuckHealingxX 1 hour ago
 				</span>
 				<h2 class="flex text-2xl align-top font-bold mb-2 text-gray-800">{{post.title}}</h2>
 				<p class="text-gray-700">{{post.content}}</p>
 			</div>
 		</a>
+>>>>>>> 5cd2c81b8926f667d0d11fa651f23551a5e124e4
```

Comment:
What happened here?

Reply:
Might need to add the post info in as it doesn't have post.content or the community name